fe/fe-core/src/main/java/org/apache/doris/catalog/TabletInvertedIndex.java

```diff
@@ -601,6 +601,23 @@ public void addReplica(long tabletId, Replica replica) {
         }
     }
 
+    // cloud mode, create table not need backendId, represent with -1.
+    public void addCloudReplica(long tabletId, Replica replica) {
+        long stamp = writeLock();
+        try {
+            Preconditions.checkState(tabletMetaMap.containsKey(tabletId),
+                    "cloudTablet " + tabletId + " not exists, replica " + replica.getId());
+            replicaMetaTable.put(tabletId, (long) -1, replica);
```

Comment:
try to use the original`addReplica` to reduce duplicated code?
```suggestion
            long bid = isCloudMode() ? -1 : replica.getBackendId();
            replicaMetaTable.put(tabletId, bid, replica);
```

Reply:
fix